index.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+entries:
+  whitesource-renovate:
+  - apiVersion: v1
+    appVersion: 2.1.1
+    created: "2022-09-02T03:16:45.624353619Z"
+    description: Responsive Dependency Automation
+    digest: ef524c2f4e8b3c853ad00df060c090251558f6bd4da9c4a46c2c4adcec3c6593
+    home: https://github.com/whitesource/renovate-on-prem
+    icon: https://app.renovatebot.com/images/logo.png
+    name: whitesource-renovate
+    sources:
+    - https://github.com/renovatebot/renovate
+    urls:
+    - https://whitesource.github.io/renovate-on-prem/whitesource-renovate-3.1.1.tgz
+    version: 3.1.1
```

Comment:
Is this a typo? I don't know where 2.1.1 or 3.1.1 come from, but I'm guessing 2.1.1 is old and 3.1.1 is a typo?

Reply:
the version here is based on Chart.yaml version which is currently 3.1.2